lib/CMakeLists.txt

```diff
@@ -868,6 +868,10 @@ if(WIN32)
 			OUTPUT_NAME "VCMI_lib"
 			PROJECT_LABEL "VCMI_lib"
 	)
+  if (MINGW)
+    # Link Windows system libs: WinSock2 (ws2_32) + socket extensions (mswsock) + Debug Help API (dbghelp)
+    target_link_libraries(vcmi PRIVATE ws2_32 mswsock dbghelp)
+  endif()
```

Comment:
please format with tabs just like the rest of the file

Reply:
good catch, done